src/components/AccountForceConnection.jsx

```diff
@@ -0,0 +1,19 @@
+import styles from '../styles/AccountForceConnection'
```

Comment:
Would it be possible to use the [`KonnectorSync`](https://github.com/cozy/cozy-collect/blob/master/src/components/KonnectorSync.jsx) component which seems appropriate for this case instead of create a new one juste pour this usage?